### PR TITLE
Tests for ability to ignore null values

### PR DESCRIPTION
**Pull request by spg**
This is a test for #183

### CODE REVIEW

**Comment by meriouma** [comment]
LGTM

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2014 ArcBees Inc.
```
Comment (meriouma) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java]: 2015

Comment (spg) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java]: @meriouma See https://github.com/ArcBees/gwtchosen/pull/188

**Comment by jDramaix** [comment]
LGTM

File: integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java (hunk 1 above)
Comment (christiangoudreau) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java]: Unless you're actually creating a new one...

Comment (christiangoudreau) [inline integration-test/src/main/java/com/arcbees/chosen/integrationtest/client/domain/CarBrand.java]: Don't you have: Copyright (c) $today.year by...etc.?

**Pull request merged**